community/dnscrypt-proxy/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Ian Bashford <ianbashford@gmail.com>
 # Maintainer: Ian Bashford <ianbashford@gmail.com>
 pkgname=dnscrypt-proxy
-pkgver=2.0.22
+pkgver=2.0.23
 pkgrel=0
 pkgdesc="A tool for securing communications between a client and a DNS resolver"
 url="https://dnscrypt.info"
```

Comment:
i think the openrc subpackage is automatically made into noarch by abuild so it is not required to turn it into it via ::noarch

Reply:
That is correct

```
case "$subpkgname" in
	*-doc | *-openrc | *-lang | *-lang-*) subpkgarch="noarch" ;;
	*) subpkgarch="$pkgarch" ;;
esac
```